polaris-react/src/components/DropZone/DropZone.scss

```diff
@@ -21,9 +21,7 @@ $dropzone-stacking-order: (
 }
 
 @mixin set-border-radius {
-  border-radius: calc(
-    var(--p-border-radius-1) + #{var(--p-border-width-1)} + 1px
```

Comment:
Wasn't sure why `border-width` was being added to `border-radius` plus `1px`? If it's important to see that math I can change it back but would definitely add a comment about it.